src/app/home/home.component.html

```diff
@@ -71,13 +71,32 @@ <h5>${{getProgress()}} of ${{getGoal()}} donated.</h5>
 
     <!--HOME PAGE CONTENT GOES HERE-->
     <mat-sidenav-content class="main-content-container">
-        <div class="main-content">
-            <div class="payment-feed-container">
+        <ng-container *ngIf ="mainContentOpened">
+            <div class="main-content">
+                <div class="payment-feed-container">
+                </div>
             </div>
-        </div>
+        </ng-container>
+
+        <ng-container *ngIf ="altViewOpened">
+            <div class ="home-view-placeholder" >
```

Comment:
not sure if this matters, but fix the spacing here to class="..." and *ngIf="..."